package.json

```diff
@@ -37,8 +37,8 @@
     "test-node": "c8 --99 --check-coverage mocha --require ./test/bootstrap/index.js test/*.js",
     "test-chrome": "web-test-runner --playwright",
     "lint": "npm run lint:js && npm run lint:format",
-    "lint:js": "eslint lib/",
-    "lint:format": "prettier --check lib",
+    "lint:js": "eslint lib/ && prettier --check lib/",
+    "lint:format": "eslint lib/ --fix && prettier --write lib/",
```

Comment:
This means the lint script will run both of them twice. I guess you meant to also update the lint script to only call lint:js?

Maybe we can make it clearer by calling the other lint:fix instead